style.css

```diff
@@ -192,7 +197,7 @@ label {
 
 input,
 textarea {
-  width: 90%;
+  width: 100%;
```

Comment:
- [ ] As requested by the previous code reviewer, the width of the textarea field is to be the same as that of the other two input elements. This way, your design will be identical to that of the Figma design that you chose. :point_down: 

  ### Your design
  [Screencast from 06-14-2023 08:14:11 PM.webm](https://github.com/donice/Portfolio-Microverse/assets/92303891/9432a20b-e7f1-43af-bafa-bfb1caddb08e)
  
  ### Figma design
  ![image](https://github.com/donice/Portfolio-Microverse/assets/92303891/583529b1-539b-4145-ba38-6a18cb12ae7a)

- [ ] While you were working on the requested changes, your project is having horizontal overflow on mobile view so make sure that you remove this overflow, as you can see in the video above to make your project responsive :+1: